theatre/studio/src/panels/OutlinePanel/BaseItem.tsx

```diff
@@ -65,7 +65,7 @@ const Header = styled(BaseHeader)`
   // Hit zone
   &:before {
     position: absolute;
-    inset: -1px -20px;
+    inset: -1px 0 -1px 0;
```

Comment:
```suggestion
    inset: -1px 0;
```

Reply:
Also, does this need any kind of comment explaining why it wasn't working before? I suppose not, but maybe @AndrewPrifer knows if he impl'd

Reply:
I'm not sure why this had `-20px` before. Removing it (making it `inset: -1px 0;`) causes the clickable area of the outline items to match their visual size.